### PR TITLE
[zh-cn] Localize autoscaling.md

### PR DESCRIPTION
**Pull request by lbzss**
Localize content/zh-cn/docs/concepts/workloads/autoscaling.md

### CODE REVIEW

File: content/zh-cn/docs/concepts/workloads/autoscaling.md [hunk 1]
```diff
@@ -0,0 +1,201 @@
+---
+title: 工作负载自动扩展
+description: >-
+  通过自动扩展，你可以用某种方式自动更新你的工作负载。在面对资源需求变化的时候可以使你的集群更灵活、更高效。
+content_type: concept
+weight: 40
+---
+<!-- ---
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/autoscaling.md:8]:
```suggestion
<!--
```

File: content/zh-cn/docs/concepts/workloads/autoscaling.md [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+title: Autoscaling Workloads
+description: >-
+  With autoscaling, you can automatically update your workloads in one way or another. This allows your cluster to react to changes in resource demand more elastically and efficiently.
+content_type: concept
+weight: 40
+--- -->
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/autoscaling.md:14]:
```suggestion
-->
```

File: content/zh-cn/docs/concepts/workloads/autoscaling.md [hunk 3]
```diff
@@ -0,0 +1,201 @@
+---
+title: 工作负载自动扩展
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/autoscaling.md:2]:
```suggestion
title: 自动扩缩工作负载
```

File: content/zh-cn/docs/concepts/workloads/autoscaling.md [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+description: >-
+  通过自动扩展，你可以用某种方式自动更新你的工作负载。在面对资源需求变化的时候可以使你的集群更灵活、更高效。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/autoscaling.md:4]:
```suggestion
  通过自动扩缩容，你可以用某种方式自动更新你的工作负载。在面对资源需求变化的时候可以使你的集群更灵活、更高效。
```

**Review by windsonsea — COMMENTED** [review]
scale 可以参考 [glossary](https://kubernetes.io/zh-cn/docs/contribute/localization_zh/#glossary)。

**Comment by windsonsea** [comment]
Please squash your commit into a single one.

File: content/zh-cn/docs/concepts/workloads/autoscaling.md [hunk 5]
```diff
@@ -0,0 +1,297 @@
...
+
+<!--
+- Learn more about scaling horizontally
+  - [Scale a StatefulSet](/docs/tasks/run-application/scale-stateful-set/)
+  - [HorizontalPodAutoscaler Walkthrough](/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough/)
+- [Resize Container Resources In-Place](/docs/tasks/configure-pod-container/resize-container-resources/)
+- [Autoscale the DNS Service in a Cluster](/docs/tasks/administer-cluster/dns-horizontal-autoscaling/)
+-->
+- 了解有关横向扩缩容的更多信息
+  - [扩缩 StatefulSet](/docs/tasks/run-application/scale-stateful-set/)
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/autoscaling.md:294]:
```suggestion
  - [扩缩 StatefulSet](/zh-cn/docs/tasks/run-application/scale-stateful-set/)
```

File: content/zh-cn/docs/concepts/workloads/autoscaling.md [hunk 6]
```diff
@@ -0,0 +1,297 @@
...
+  - [HorizontalPodAutoscaler 演练](/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough/)
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/autoscaling.md:295]:
```suggestion
  - [HorizontalPodAutoscaler 演练](/zh-cn/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough/)
```

File: content/zh-cn/docs/concepts/workloads/autoscaling.md [hunk 7]
```diff
@@ -0,0 +1,297 @@
...
+- [调整分配给容器的 CPU 和内存资源](/docs/tasks/configure-pod-container/resize-container-resources/)
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/autoscaling.md:296]:
```suggestion
- [调整分配给容器的 CPU 和内存资源](/zh-cn/docs/tasks/configure-pod-container/resize-container-resources/)
```

File: content/zh-cn/docs/concepts/workloads/autoscaling.md [hunk 8]
```diff
@@ -0,0 +1,297 @@
...
+- [自动扩缩集群 DNS 服务](/docs/tasks/administer-cluster/dns-horizontal-autoscaling/)
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/autoscaling.md:297]:
```suggestion
- [自动扩缩集群 DNS 服务](/zh-cn/docs/tasks/administer-cluster/dns-horizontal-autoscaling/)
```

File: content/zh-cn/docs/concepts/workloads/autoscaling.md [hunk 9]
```diff
@@ -0,0 +1,297 @@
...
+If scaling workloads isn't enough to meet your needs, you can also scale your cluster infrastructure itself.
+-->
+如果扩缩容工作负载无法满足你的需求，你也可以扩缩容集群基础设施本身。
+
+<!--
+Scaling the cluster infrastructure normally means adding or removing {{< glossary_tooltip text="nodes" term_id="node" >}}.
+This can be done using one of two available autoscalers:
+-->
+扩缩容集群基础设施通常是指增加或移除{{< glossary_tooltip text="节点" term_id="node" >}}。
+这可以通过以下两种扩缩容器中的任意一种实现：
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/autoscaling.md:272]:
```suggestion
这可以通过以下两种自动扩缩器中的任意一种实现：
```
- auto 也可以翻出来
- scale 可以上下文统一成 扩缩，参考 [扩缩 StatefulSet](https://deploy-preview-45206--kubernetes-io-main-staging.netlify.app/zh-cn/docs/tasks/run-application/scale-stateful-set/)

File: content/zh-cn/docs/concepts/workloads/autoscaling.md [hunk 10]
```diff
@@ -0,0 +1,297 @@
...
+object that manages a set of Pods (for example a
+{{< glossary_tooltip text="Deployment" term_id="deployment" >}}.
+-->
+在 Kubernetes 中**自动扩缩容**的概念是指自动更新管理一组 Pod 的能力（例如
+{{< glossary_tooltip text="Deployment" term_id="deployment" >}}）。
+
+<!--
+### Scaling workloads horizontally
+-->
+### 水平扩缩容工作负载
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/autoscaling.md:86]:
```suggestion
### 水平扩缩工作负载   {#scaling-workloads-horizontally}
```

File: content/zh-cn/docs/concepts/workloads/autoscaling.md [hunk 11]
```diff
@@ -0,0 +1,297 @@
...
+- **Horizontal scaling**: [Running multiple instances of your app](/docs/tutorials/kubernetes-basics/scale/scale-intro/)
+- **Vertical scaling**: [Resizing CPU and memory resources assigned to containers](/docs/tasks/configure-pod-container/resize-container-resources)
+-->
+- **水平扩缩容**：[运行应用程序的多个实例](/docs/tutorials/kubernetes-basics/scale/scale-intro/)
+- **垂直扩缩容**：[调整分配给容器的 CPU 和内存资源](/docs/tasks/configure-pod-container/resize-container-resources)
+
+<!--
+## Scaling workloads automatically
+-->
+## 自动扩缩容工作负载
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/autoscaling.md:68]:
```suggestion
## 自动扩缩工作负载   {#scaling-workloads-automatically}
```
Anchors are required for heading 2 and 3.

**Review by windsonsea — COMMENTED** [review]
These terms shall be distinguished:

- scale 扩缩
- autoscale 自动扩缩
- autoscaler 自动扩缩器

Refer to [some current pages](https://deploy-preview-45206--kubernetes-io-main-staging.netlify.app/zh-cn/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough/#increase-load).

**Comment by tengqm** [comment]
仍然有多处将 scaling 译为“扩展”。“扩展”对应的英文单词是“extending”、“extension”.

**Pull request merged**